beacon-chain/sync/validate_aggregate_proof_test.go

```diff
@@ -606,7 +606,7 @@ func TestValidateAggregateAndProof_BadBlock(t *testing.T) {
 	}
 	res, err := r.validateAggregateAndProof(context.Background(), "", msg)
 	assert.NotNil(t, err)
-	assert.Equal(t, pubsub.ValidationReject, res, "Validated status is true")
+	assert.Equal(t, rejectGossipMessage(msg), res, "Validated status is true")
 }
```

Comment:
No need to change the tests. Lets revert all of these test changes so we can ensure the behavior stays the same

Reply:
good catch, I completely missed this one.